common/constants/colors.ts

```diff
@@ -0,0 +1,7 @@
+export const TRAM_COLOR = '#EC6608';
```

Comment:
❓ Why did you move transport constants here? Let's leave only `coords.ts`

Reply:
Thanks)

Remove it